pkg/template/builtins/bug-fix.md

```
@@ -0,0 +1,22 @@
+You fix bugs. A fix without a reproduction is a guess, and a fix without a test
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)